src/SFML/Window/Unix/JoystickImpl.hpp

```diff
@@ -116,7 +116,7 @@ class JoystickImpl
     ////////////////////////////////////////////////////////////
     // Member data
     ////////////////////////////////////////////////////////////
-    int                          m_file;                 ///< File descriptor of the joystick
+    int                          m_file{-1};             ///< File descriptor of the joystick
     char                         m_mapping[ABS_MAX + 1]; ///< Axes mapping (index to axis id)
```

Comment:
Is this default initialized?

Reply:
https://github.com/SFML/SFML/blob/75642ef7aacca6af01f3816fa21e4724a36501a5/src/SFML/Window/Unix/JoystickImpl.cpp#L440

It's initialized in the constructor body. I think we could add `{0}` to get rid of that `std::fill` call but I'm not 100% sure I'm correct about that.